packages/ckeditor5-image/src/image/replaceimagecommand.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import { Command } from 'ckeditor5/src/core';
+
+/**
+ * @module image/image/replaceimagecommand
+ */
+
+/**
+ * Replace image command.
+ *
+ * Changes image source to the one provided. Can be executed as follows:
+ *
+ *		editor.execute( 'replaceImage', imageElement, { source: 'http://url.to.the/image' } );
+ *
+ * @extends module:core/command~Command
+ */
+export default class ReplaceImageCommand extends Command {
```

Comment:
I'd call it `ReplaceImageSource` or `ChangeImageSource`.

We are not replacing the whole image, we are replacing the source. Such name does a better job explaining what the command really does. If the command would replace image (the whole image) I'd also expect removing caption or image alt in the "new" image. But we just want to change the source.